jans-lock/cedarling/cedarling/src/log/log_strategy.rs

```diff
@@ -67,3 +67,203 @@ impl LogStorage for LogStrategy {
         }
     }
 }
+
```

Comment:
Is it possible to add the test cases in a separate file(s) in `jans-lock/cedarling/cedarling/src/log/test` directory.

Reply:
Thanks, unit tests for `LogStrategy` moved to `log/test` module
I am not sure if we need to move others unit tests. Because It will be a mess. And actually, `LogStrategy` unit tests duplicate some parts of other unit tests.